docs/resources/obs_bucket_replication.md

```diff
@@ -0,0 +1,111 @@
+---
+subcategory: "Object Storage Service (OBS)"
+---
+
+# huaweicloud_obs_bucket_replication
+
+Manages an OBS bucket **Cross-Region Replication** resource within HuaweiCloud.
+
+-> **NOTE:** The source bucket and destination bucket must belong to the same account. More cross-Region replication
+constraints see [Cross-Region replication](https://support.huaweicloud.com/intl/en-us/ugobs-obs/obs_41_0034.html)
+
+## Example Usage
+
+### Replicate all objects
+
+```hcl
+variable "bucket" {}
+variable "destination_bucket" {}
+variable "agency" {}
+
+resource "huaweicloud_obs_bucket_replication" "test" {
+  bucket             = var.bucket
+  destination_bucket = var.destination_bucket
+  agency             = var.agency
+
+  rule {}
```

Comment:
can we delete this line?

Reply:
This line can be removed when create. I will remove it.